src/Http/Session/DatabaseSession.php

```diff
@@ -57,7 +57,7 @@ public function __construct(array $config = [])
         $tableLocator = $this->getTableLocator();
 
         if (empty($config['model'])) {
-            $config = $tableLocator->exists('Sessions') ? [] : ['table' => 'sessions'];
+            $config = $tableLocator->exists('Sessions') ? [] : ['table' => 'sessions', 'allowFallbackClass' => true];
             $this->_table = $tableLocator->get('Sessions', $config);
```

Comment:
The config is ignored if the table exists, right? Do we need this extra logic?

Reply:
No, the config is not ignored. If an alias is already present is the registry and you call get() again for it with a different config then it will throw an exception. So the extra logic is necessary.

Reply:
that's right, my mistake.